docs/versioned_docs/version-v3.2.0/02-Features/running-integration-tests.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Running integration tests with Arcus"
+layout: default
+---
+
+## Running integration tests with Arcus
+
+![](https://img.shields.io/badge/Available%20starting-v3.0-green?link=https://github.com/arcus-azure/arcus.eventgrid/releases/tag/v3.0.0)
+
+We provide some minimal testing infrastructure that allows you to run integration tests on top of Azure Event Grid.
+
+Easy to start:
+```shell
+> Install-Package Arcus.EventGrid.Testing
+```
+
+Here is what this article will cover:
+
+- [Running integration tests with Arcus](#running-integration-tests-with-arcus)
+  - [Receiving events in your tests](#receiving-events-in-your-tests)
+    - [Azure Service Bus test example](#azure-service-bus-test-example)
+    - [Available functionality for receiving events](#available-functionality-for-receiving-events)
+  - [Azure infrastructure](#azure-infrastructure)
+  - [Troubleshooting tests](#troubleshooting-tests)
+    - [Use logging](#use-logging)
+    - [Keeping a test subscription on the topic](#keeping-a-test-subscription-on-the-topic)
+
+### Receiving events in your tests
+By using the `ServiceBusEventConsumerHost` you can subscribe to Azure Event Grid events on a custom topic and consume them in your tests.
+
+As requests are flowing in asynchronously, we provide the capability to retry the looking for an event which is using an exponential back off.
```

Comment:
What is meant by 'the looking for an event' ?  Do you mean the 'lookup of an event' ?  Or the receival of an event ?

Reply:
It's probably 'lookup' bc the events are already stored async here, and the consumer has to provide the necessary query/filter to 'look them up'.

Reply:
Done in https://github.com/arcus-azure/arcus.eventgrid/pull/232/commits/b8a4cd16ea2e9205531a3c34ff29c01f30d9324e